.eslintignore

```diff
@@ -4,3 +4,4 @@ test
 *.gz
 *.sql
 *.zst
+services/**/data/**
```

Comment:
eslint complains about `blockchain-app-registry/data/app-registry/validation` directory depencencies. That is why added this exclusion. It can be applied to the specific folder, but thought ideally data won't need linting anyway.